galaxychop/core.py

```diff
@@ -219,87 +213,43 @@ def values(self, star=True, gas=True, dm=True):
         ------
         X = 'np.ndarray(n,7)'
             2D array where each file it is a diferen particle and
-            each colum it is a parameter of the particles (x,y,z,vx,vy,vz,m)
+            each colum it is a parameter of the particles (E_star, eps, eps_r)
         y = 'np.ndarray(n)'
             1D array where is identified the nature of each particle
             0 = star, 1 = gas and 2 = dark mather
         """
-        arr_ = self.arr_
-
-        X = np.empty((0, 7))
+        X = np.empty((0, 3))
         y = np.empty(0, int)
 
         if star:
-            n_s = len(arr_.x_s)
+            n_s = len(self.paramcirc[1])
 
             X_s = np.hstack(
                 (
-                    arr_.x_s.reshape(n_s, 1),
-                    arr_.y_s.reshape(n_s, 1),
-                    arr_.z_s.reshape(n_s, 1),
-                    arr_.vx_s.reshape(n_s, 1),
-                    arr_.vy_s.reshape(n_s, 1),
-                    arr_.vz_s.reshape(n_s, 1),
-                    arr_.m_s.reshape(n_s, 1),
+                    self.paramcirc[0].reshape(n_s, 1),
+                    self.paramcirc[1].reshape(n_s, 1),
+                    self.paramcirc[2].reshape(n_s, 1),
                 )
             )
             y_s = np.zeros(n_s)
 
             X = np.vstack((X, X_s))
             y = np.hstack((y, y_s))
 
-        if gas:
-            n_g = len(arr_.x_g)
-
-            X_g = np.hstack(
-                (
-                    arr_.x_g.reshape(n_g, 1),
-                    arr_.y_g.reshape(n_g, 1),
-                    arr_.z_g.reshape(n_g, 1),
-                    arr_.vx_g.reshape(n_g, 1),
-                    arr_.vy_g.reshape(n_g, 1),
-                    arr_.vz_g.reshape(n_g, 1),
-                    arr_.m_g.reshape(n_g, 1),
-                )
-            )
-            y_g = np.zeros(n_g)
-
-            X = np.vstack((X, X_g))
-            y = np.hstack((y, y_g))
-
-        if dm:
-            n_dm = len(arr_.x_dm)
-
-            X_dm = np.hstack(
-                (
-                    arr_.x_dm.reshape(n_dm, 1),
-                    arr_.y_dm.reshape(n_dm, 1),
-                    arr_.z_dm.reshape(n_dm, 1),
-                    arr_.vx_dm.reshape(n_dm, 1),
-                    arr_.vy_dm.reshape(n_dm, 1),
-                    arr_.vz_dm.reshape(n_dm, 1),
-                    arr_.m_dm.reshape(n_dm, 1),
-                )
-            )
-            y_dm = np.zeros(n_dm)
-
-            X = np.vstack((X, X_dm))
-            y = np.hstack((y, y_dm))
-
         return X, y
 
     @property
     def energy(self):
         """
-        Energy calculation.
+        Specific energy calculation.
 
-        Calculate kinetic and potencial energy of dark matter,
-        star and gas particles.
+        Calculate the specific kinetic and specific potencial energy
```

Comment:
I would change this sentence by: "Calculate the specific kinetic and potential energies"